ietf/topics/templates/topics/topic_index_page.html

```diff
@@ -3,38 +3,45 @@
 {% load wagtailimages_tags wagtailcore_tags %}
 
 {% block main_content %}
-  <div class="body-white rule-blue">
-    <div class="sml-container center">
-      {% include 'includes/breadcrumbs.html' %}
-      <h1 class="center-div">{{ self.title }}</h1>
-      <p class="intro center-div">{{ self.introduction }}</p>
-      {% include 'includes/social_fields.html' %}
+<main role="main">
+
+  <div class="bg-white border-bottom pb-3">
+    <div class="container">
+      <div class="u-max-text-width">
+
+        {% include 'includes/breadcrumbs.html' %}
+        <h1>{{ self.title }}</h1>
+        <p class="lead">{{ self.introduction }}</p>
+        
+        {% include 'includes/social_fields.html' %}{# TODO??? keep?/change?move? #}
+      </div>
     </div>
   </div>
-
-  {# primary topics #}
-  <div class="body-colour">
-    <ul class="topic-list">
+  
+  <div class="container">
+    <ul class="topic-list list-unstyled mt-5">
       {% for primary_topic in self.primary_topics %}
-        <li>
-            <a href="{% pageurl primary_topic %}">
-                <div class="container body">
-                    {% if primary_topic.feed_image %}
-                        <div class="img-holder">
-                            {% image primary_topic.feed_image original as feed_image %}
-                            <img src="{{ feed_image.url }}" alt="{{ primary_topic.feed_image.title }}">
-                        </div>
-                    {% endif %}
-                    <div class="text-holder">
-                        <h2>{{ primary_topic.title }}</h2>
-                        <p>{{ primary_topic.feed_text }}</p>
-                    </div>
-                </div>
-            </a>
-        </li>
+      <li class="border-bottom pb-5 mb-5 border-primary">
+        <div class="row d-flex flex-column flex-md-row align-items-md-center w-100">
```

Comment:
Should the `topic-list` be applied to this element instead so the borders on the `li` can be full width?